openslides_backend/action/motion_category/number_motions.py

```diff
@@ -0,0 +1,225 @@
+from typing import Dict, List, Optional
+
+from ...models.models import Motion, MotionCategory
+from ...services.datastore.commands import GetManyRequest
+from ...shared.exceptions import ActionException
+from ...shared.patterns import Collection, FullQualifiedId
+from ..base import ActionPayload
+from ..default_schema import DefaultSchema
+from ..generics import UpdateAction
+from ..register import register_action
+
+
+@register_action("motion_category.number_motions")
+class MotionCategoryNumberMotions(UpdateAction):
+    """
+    Action to number motions for a category.
+    """
+
+    model = Motion()
+    schema = DefaultSchema(MotionCategory()).get_default_schema(
+        title="Number motions of a motion category.",
+        description="An object containing an array of main category id.",
+        required_properties=["id"],
+    )
+
+    def get_updated_instances(self, payload: ActionPayload) -> ActionPayload:
+        for instance in payload:
+            self.init_memory(instance["id"])
```

Comment:
is it ensured, that the id is valid somewhere? Maybe write a test with an invalid id to see that it is handeled properly.

Reply:
Added a test.